tools/ci_build/github/android/build_aar_package.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+
+from pathlib import Path
+
+REPO_ROOT = Path(__file__).parents[4]
+BUILD_PY = REPO_ROOT / "build.py"
+JAVA_ROOT = REPO_ROOT / "src" / "java"
+sys.path.append(str(REPO_ROOT / "tools" / "python"))
+import util  # REPO_ROOT/tools/python/util noqa: E402
+
+# Default to only build 64-bit ABIs. TBD if we need to support any 32-bit ABIs.
+# DEFAULT_BUILD_ABIS = ["armeabi-v7a", "arm64-v8a", "x86", "x86_64"]
+DEFAULT_BUILD_ABIS = ["arm64-v8a", "x86_64"]
+
+# Match onnxruntime and use NDK API 21 by default
+# It is possible to build from source for Android API levels below 21, but it is not guaranteed.
+# As level 21 is Android 5.0 from 2014, it's highly unlikely that we will need to support anything lower.
+DEFAULT_ANDROID_MIN_SDK_VER = 21
```

Comment:
This feels really old, but as we're not dependent on any specific Android functionality it shouldn't matter.